buildx.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
@quincycheng where is this script used? is it for dev purposes? can you add a usage guide?

Reply:
+1 on this comment - can we just add building this script to build.sh? what's the benefit of having a whole separate script?